descartes_moveit/include/descartes_moveit/jaco3_moveit_state_adapter.h

```diff
@@ -72,6 +73,7 @@ class Jaco3MoveitStateAdapter : public descartes_moveit::MoveitStateAdapter
   std::string octomap_link_ = "<octomap>";
   std::vector<std::string> collision_arm_links_ = {"half_arm_2_link", "forearm_link"};
   std::vector<std::string> collision_robot_links_ = {"tower_link", "camera_box"};
+  std::vector<std::string> collision_octomap_links_ = {"sprayer"};
```

Comment:
Will we have to change and recompile each time we add a tool or change the name? Would it be cleaner to pull this out into a parameter?

Reply:
`peanut_motion_planning` exposes a service that can update the state adapters `collision_arm_links_` and `collision_robot_links_`. In hindsight, using parameters would have been easier